webonary-cloud-api/lambda/getDictionary.ts

```diff
@@ -81,9 +83,19 @@ export async function handler(event: APIGatewayEvent): Promise<APIGatewayProxyRe
   }
 
   // TODO: Populate these during dictionary upload
+  const entriesCollection = db.collection(dbCollectionEntries(dictionaryId));
+
+  // get total entries and semantic domains used in senses
+  [dbItem.mainLanguage.entriesCount, dbItem.semanticDomainAbbreviationsUsed] = await Promise.all([
```

Comment:
Are these queries expensive? Maybe consider storing this in the dictionary entry during upload?

Reply:
Not too expensive. And dictionary info is cached in Webonary until a new dictionary is used. If we store them during dictionary upload, we still need to do this here, for legacy dictionaries, unless we do a data migration.

Once this project becomes stable, I think that would be a good time to do this logic during upload